neverpile-eureka-application/pom.xml

```diff
@@ -41,8 +41,8 @@
       <artifactId>neverpile-eureka-security-oauth2</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.neverpile.eureka</groupId>
-      <artifactId>neverpile-openapi</artifactId>
+      <groupId>com.neverpile.commons</groupId>
+      <artifactId>neverpile-commons-swagger-ui</artifactId>
```

Comment:
`neverpile-commons-openapi` is included in `neverpile-commons-swagger-ui`, good to know.

Reply:
Yes, `neverpile-commons-openapi` is also a dependency of `neverpile-eureka-core`, so you basically never need to declare it.